client/homebrew/editor/snippetbar/snippetbar.jsx

```diff
@@ -60,10 +60,35 @@ const Snippetbar = createClass({
 		});
 	},
 
+	toTop : function(){
+		this.props.sourceJump(0);
+		this.props.brewJump(1);
+	},
+
+	moveBrew : function(){
```

Comment:
If these functions are just wrappers to call another function with no change, we probably don't need this and can call `this.props.brewJump()` directly from the button.

Reply:
They are wrappers at the moment; if further development doesn't discover a need for any additional functionality here, then my plan is to call it directly.